torch/_inductor/codegen/wrapper_fxir.py

```diff
@@ -486,10 +487,38 @@ def _generate_triton_call(self, line: WrapperLine) -> None:
         call_kwargs = dict(zip(tuner.triton_meta["signature"], call_args))
         call_kwargs.update(config.kwargs)
 
+        def replace_floor_div(arg: sympy.Expr) -> sympy.Expr:
```

Comment:
Use `expr` to make it more readable?
```suggestion
        def replace_floor_div(expr: sympy.Expr) -> sympy.Expr:
```

Reply:
Good idea. Changed in [c195087](https://github.com/pytorch/pytorch/pull/153230/commits/c1950871e16364c4f8ee034cd67f1f8e53589982).